python/py-pyhamcrest/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pyhamcrest
+version             1.9.0
+platforms           darwin
+license             BSD
+maintainers         {mcalhoun @MarcusCalhoun-Lopez} openmaintainer
+
+description         Hamcrest framework for matcher objects
+long_description    ${description}
+
+homepage            https://github.com/hamcrest/PyHamcrest
+master_sites        pypi:P/PyHamcrest
+distname            PyHamcrest-${version}
+
+checksums           rmd160  f5fa85ec3e29c3ec41e28ae8e690700fbf2025a1 \
+                    sha256  8ffaa0a53da57e89de14ced7185ac746227a8894dbd5a3c718bf05ddbd1d56cd \
+                    size    376623
+
+python.versions     27 35 36 37
```

Comment:
py34-twisted will break as py34-pyhamcrest is missing.

Reply:
Sorry I didn't notice there are already discussions about this. I prefer to drop py34-scrapy, too.